app/frontend/components/pages/login/ledgerTransportationSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import {h} from 'preact'
+import SearchableSelect from '../../common/searchableSelect'
+import {LedgerTransportType} from '../../../../frontend/types'
+import styles from './ledgerTransportationSelect.module.scss'
+
+interface Props {
+  onSelect: (ledgerTransportType: LedgerTransportType) => void
+}
+
+const LedgerTransportationSelect = ({onSelect}: Props) => {
+  const dropdownAssetItems = [
+    LedgerTransportType.DEFAULT,
```

Comment:
I'd consider moving the "smart" transport selection logic to the app (or exposing the function returning the enum from ledger crypto provider, and we would actually know what that "default" transport is) - this would allow us to display appropriate guidance to the users based on the transport chosen

Reply:
Refactored.